lib/ledger_sync/adaptors/netsuite_rest/token.rb

```diff
@@ -0,0 +1,162 @@
+# frozen_string_literal: true
+
+# ref: https://docs.oracle.com/cloud/latest/netsuitecs_gs/NSATH/NSATH.pdf
+# ref: Search "The Signature for Web Services and RESTlets" in NetSuite documentation (must have an account)
+module LedgerSync
+  module Adaptors
+    module NetSuiteREST
+      class Token
+        DEFAULT_SIGNATURE_METHOD = 'HMAC-SHA256'
```

Comment:
This is the one that works I'm assuming?

Reply:
Yes.  I believe both work `HMAC-SHA256` and `HMAC-SHA1`